client/css/core-general.styl

```diff
@@ -284,28 +284,25 @@ a .access-key
     /*background-image: attr(data-src url)*/ /* not available yet */
     vertical-align: middle
     background-repeat: no-repeat
-    background-size: cover
+    background-size: contain
     background-position: center
     display: inline-block
+    overflow: hidden
     width: 20px
     height: 20px
     &.empty
         background-image:
-            linear-gradient(45deg, $transparency-grid-square-color 25%, transparent 25%),
-            linear-gradient(-45deg, $transparency-grid-square-color 25%, transparent 25%),
-            linear-gradient(45deg, transparent 75%, $transparency-grid-square-color 75%),
-            linear-gradient(-45deg, transparent 75%, $transparency-grid-square-color 75%)
-        background-position: 0 0, 0 10px, 10px -10px, -10px 0px
-        background-repeat: repeat
-        background-size: 20px 20px
-    img
+            repeating-linear-gradient(45deg, $window-color, $window-color 10px, #e6e6e6 10px, #e6e6e6 20px)
+    img, video
```

Comment:
I assume these changes should be taken over https://github.com/rr-/szurubooru/pull/642/files#diff-965c9ad95631d07c4423bbe276db1d09308e9d790f997c4232f3896b6405b407 right?